core/src/test/java/org/web3j/contracts/token/ERC20BasicInterface.java

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
Please can you remove the copyright from these files?

Reply:
Oh shoot, absolutely.  IDE user error on my part.